Lib/fontTools/ttLib/ttFont.py

```diff
@@ -673,7 +678,7 @@ def getTableData(self, tag):
 		else:
 			raise KeyError(tag)
 
-	def getGlyphSet(self, preferCFF=True):
+	def getGlyphSet(self, preferCFF=True, location=None, normalized=False):
 		"""Return a generic GlyphSet, which is a dict-like object
```

Comment:
docstring could mention the new location and normalized parameters, and the fact that currently this only works for TrueType-flavored variable fonts. Maybe you could add some TODO or file an issue for CFF2 support